src/components/common/button/index.js

```diff
@@ -0,0 +1,40 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import React from 'react'
+import * as ReactRouter from 'react-router-dom'
+
+import Flex from 'components/common/flex'
+
+import style from './index.style.js'
+
+class Button extends React.Component {
+  render() {
+    return (
+      <Flex
```

Comment:
So I've read about using `<button>` vs `<div>` for clickable elements and it looks like the latter is a bad practice. There might be problems with SEO, automated tests and accessibility, so I guess we should at least try to use buttons (or `<a>`'s) where possible.

I've added a global style overwrite that resets all buttons and should make them render same way as `<div>`s

Let's change this back to div 😅

Reply:
replaced Flex with button